nodes/wot-discovery.js

```diff
@@ -22,6 +24,11 @@ module.exports = function (RED) {
             coapAddresses = _getCoapAddresses(config);
         }
 
+        if(coreURI){
+            if(config.useCoap){
+            }
+        }
+
```

Comment:
Are these lines still needed?

Reply:
No they ares useless. I removed them in dff99200e621c94ca3d7ee4f954569419729f2c9